lineman/app/components/thread_page/proposal_expanded/proposal_expanded.haml

```diff
@@ -7,8 +7,9 @@
     %span.time-ago>
       {{proposal.createdAt | timeFromNowInWords}}
     %span{aria-hidden: 'true'}·
-    %closing_in{time: 'proposal.closingAt' }>
-    .badge.pull-right.cuke-proposal-closed-badge{ng-show: '!proposal.isActive'} Closed
+    %closing_in{ng-if: 'proposal.isActive()', time: 'proposal.closingAt' }>
+    %closing_in{ng-if: '!proposal.isActive()', time: 'proposal.closedAt' }>
+    .badge.pull-right.cuke-proposal-closed-badge{ng-show: '!proposal.isActive()'} Closed
```

Comment:
Nice catch.  

James and Hannah this might be a good time for me to say: We've already had feedback that the "Common" section in the translations file is sometimes causing problems. I think that keeping everything namespaced and allowing duplicates is a much better approach - Transifex detects and offers to autocomplete duplicates, meaning that translators get ease and flexibility.

IE: in this case the translation key should be `proposal_expanded.closed_badge` rather than `common.closed`

Also @HSalmon can you please remove the .cuke- class?